libraries/pbrlib/pbrlib_defs.mtlx

```diff
@@ -404,4 +423,40 @@
     <output name="extinction" type="color3" />
   </nodedef>
 
+  <!--
+    Node: <deon_hair_absorption_from_melanin>
+    Calculates hair absorption from melanin parameters.
+  -->
+  <nodedef name="ND_deon_hair_absorption_from_melanin" node="deon_hair_absorption_from_melanin" nodegroup="pbr" doc="Calculates hair absorption from melanin parameters.">
+    <input name="melanin_concentration" type="float" value="0.25" />
+    <input name="melanin_redness" type="float" value="0.5" />
+    <input name="eumelanin_color" type="color3" value="0.657704, 0.498077, 0.254107" colorspace="lin_rec709" doc="constant from d'Eon et al. 2011, converted to color via exp(-c)" uiadvanced="true"/>
+    <input name="pheomelanin_color" type="color3" value="0.829444, 0.67032, 0.349938" colorspace="lin_rec709" doc="constant from d'Eon et al. 2011, converted to color via exp(-c)" uiadvanced="true"/>
+    <output name="absorption" type="vector3" />
+  </nodedef>
+
+  <!--
+    Node: <chiang_hair_absorption_from_color>
+    Calculates hair absorption from a color.
+  -->
+  <nodedef name="ND_chiang_hair_absorption_from_color" node="chiang_hair_absorption_from_color" nodegroup="pbr" doc="Calculates hair absorption from a color.">
+    <input name="color" type="color3" value="1.0, 1.0, 1.0" />
+    <input name="azimuthal_roughness" type="float" value="0.2" />
+    <output name="absorption" type="vector3" />
+  </nodedef>
+
+  <!--
+    Node: <chiang_hair_roughness>
+    Calculates hair roughness for R, TT and TRT component.
+  -->
+  <nodedef name="ND_chiang_hair_roughness" node="chiang_hair_roughness" nodegroup="pbr" doc="Calculates hair roughness for R, TT and TRT component.">
+    <input name="longitudinal" type="float" value="0.1" />
+    <input name="azimuthal" type="float" value="0.2" />
+    <input name="scale_TT" type="float" value="0.5" />
```

Comment:
we could tag scale_TT + scale_TRT as "uiadvanced"
ie...
`<input name="scale_TT" type="float" value="0.5" uiadvanced="true"/>`
`<input name="scale_TRT" type="float" value="2.0" uiadvanced="true"/>`

Reply:
sure, added those.